DataFormats/EgammaCandidates/src/classes_def.xml

```diff
@@ -321,9 +321,15 @@
   </class>
   <class name="edm::helpers::KeyVal<edm::RefProd<std::vector<reco::Photon> >,edm::RefProd<edm::OwnVector<reco::Candidate,edm::ClonePolicy<reco::Candidate> > > >" />
   <class name="edm::Wrapper<edm::AssociationMap<edm::OneToOne<std::vector<reco::Photon>,edm::OwnVector<reco::Candidate,edm::ClonePolicy<reco::Candidate> >,unsigned int> > >" />
- 
+
   <class name="edm::Ptr<reco::Photon>" />
   <class name="edm::PtrVector<reco::Photon>" />
-  
-</lcgdict>
 
+  <class name="aod::AODHIPhoton" ClassVersion="10">
```

Comment:
this can start with 2
(0 is special, and I can't recall if 1 is special as well)